dm/worker/log.go

```diff
@@ -505,3 +525,36 @@ func CloneTaskLog(log *pb.TaskLog) *pb.TaskLog {
 func whetherNil(handler interface{}) bool {
 	return handler == nil || reflect.ValueOf(handler).IsNil()
 }
+
+// clearByPrefix clears all keys with the specified prefix.
+func clearByPrefix(txn *leveldb.Transaction, prefix []byte) error {
```

Comment:
why not use `Deleter`

Reply:
`Deleter` can't `write` by _batch_, change to use `Deleter` and remove _batch_?

Reply:
If so, I suggest that you try to unify as much as possible andyou can change the definition of interface. I'm not sure what leveldb's performance would be in bulk when deleting most of the data in a transaction.

Reply:
what about it?

Reply:
added two methods (`Write`, `NewIterator`) into `Putter`, `Getter` and `Deleter` in https://github.com/pingcap/dm/pull/187/commits/1cf143fd09ddd4d274a8b20a4656d31b71ae881b.